apps/poker-roll/src/app/activeSessionComponent.jsx

```diff
@@ -37,7 +34,6 @@ const ActiveSession = (props) => {
 };
 const ActiveSessionContainer = () => {
   const sessionsQuery = useGetAllSessions();
-  console.log(sessionsQuery.data);
   return sessionsQuery.data?.map((session) => {
```

Comment:
Seeing an error in the console here: `Warning: Each child in a list should have a unique "key" prop.` https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key

Reply:
i'm not getting this error here after adding a key, but i'm getting the same error in the Past Session map function